chai/chai.d.ts

```diff
@@ -14,6 +14,12 @@ declare module chai {
 
     export function expect(target: any, message?: string): Expect;
 
+    // The should style allows for the same chainable assertions as the expect interface, by extending any object to have 'should' property.
+    export function should(): Expect;
```

Comment:
FWIW, the object returned by should() is not an Expect. The "should" object only has a handful of methods, and methods like should.equal(actual, expected) take both the actual and the expected value. See:

https://github.com/chaijs/chai/blob/master/lib/chai/interface/should.js

My PR #4155 reflects that.